spec/api-server.spec.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+describe("ApiServer", () => {
+    var ApiServer, webServerMock;
+
+    beforeEach(() => {
+        ApiServer = require("../lib/api-server");
+        webServerMock = jasmine.createSpyObj("webServerMock", [
+            "configure",
+            "addRoute",
```

Comment:
It is adding "spec/api-server.spec.js" though that's really not part of your commits.  This is an example of why I'd love to have you base work off of "master" in the future.